tests/FSharp.Compiler.ComponentTests/Debugger/PortablePdbs.fs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.Debugger
+
+open Xunit
+open FSharp.Test.Compiler
+open System.Reflection.Metadata
+
+module PortablePdbs =
```

Comment:
Can we also test opening user namespaces defined with namespace.  and modules.  as well as open before modules and after modules.